### PR TITLE
devices/yuga: Add content; replace all template texts

### PR DESCRIPTION
**Pull request by LNJ2**

### CODE REVIEW

File: supplementary/devices/yuga.rst [hunk 1]
```diff
@@ -1,22 +1,22 @@
 
 Yuga - Sony Xperia Z
-=====================
+====================
 
-Here only one or two sentences. What makes the device special? Is it a reference device? 
+There exsits basic mainline kernel support added by Sony. This possibly could
```
Comment (JBBgameich) [inline supplementary/devices/yuga.rst]: exsits -> exists

File: supplementary/devices/yuga.rst [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
 Status
 ------
 
 Halium
 ^^^^^^
 
-There is a **working Halium 7.1** made by `LNJ2 <https://github.com/LNJ2>`_ and the according Halium/Projectmanagement issue is `here <https://github.com/Halium/projectmanagement/issues/19>`_.
+There is a **working Halium 7.1** made by `LNJ <https://github.com/LNJ2>`_ and
+the according Halium/Projectmanagement issue is `here <https://github.com/Halium/projectmanagement/issues/19>`_.
```
Comment (UniversalSuperBox) [inline supplementary/devices/yuga.rst]: When making links, make the link text explanatory. This is for a couple reasons:

* Sphinx does not like it when multiple links are named the same thing. Making links with names like "here" makes this more likely.
* Links should describe their target in general

File: supplementary/devices/yuga.rst [hunk 3]
```diff
@@ -25,45 +25,53 @@ The following entries are just a placeholder, exactly as this sentence.
...
 
 Developer Info
 ^^^^^^^^^^^^^^
 
-Some devices show strange behaviour of some kind, try to find this (for example in the xda-developers forum) and document it
+There's no strange behaviour, except the way to get into fastboot or recovery.
 
-Usefull Ressources
-^^^^^^^^^^^^^^^^^^
+Useful Ressources
```
Comment (UniversalSuperBox) [inline supplementary/devices/yuga.rst]: Resources

File: supplementary/devices/yuga.rst [hunk 4]
```diff
@@ -25,45 +25,53 @@ The following entries are just a placeholder, exactly as this sentence.
...
 Kernel & Hardware
 ^^^^^^^^^^^^^^^^^
 
-Mainline (vX.Y.Z as of writing)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Mainline (v4.15-rc9 as of writing)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The kernel will start and there is already UART and USB debugging implemented
+for this device. There also has been worked on riva, which is AFAIK for
```
Comment (UniversalSuperBox) [inline supplementary/devices/yuga.rst]: Maybe add a link to 'riva' instead of using "As Far as I Know". Only say what you know is true.

**Pull request merged**